test/living-html/htmlcanvaselement.js

```diff
@@ -117,6 +117,34 @@ exports["toDataURL should work (when the canvas npm package is provided) (GH-102
   t.done();
 };
 
+exports["loading an image and drawing it into the canvas should produce the expected result"] = t => {
+  if (!isCanvasInstalled(t)) {
+    return;
+  }
+
+  const document = jsdom.jsdom("<canvas width='168' height='168'></canvas>",
+      { features: { FetchExternalResources: ["img"] } });
+  const window = document.defaultView;
+  const canvas = document.querySelector("canvas");
+  const ctx = canvas.getContext("2d");
+  const image = new window.Image();
+  image.src = "file://" + path.resolve(__dirname, "files/image.png");
```

Comment:
Need to use the test util toFileUrl here to get this to work on Windows (where paths are backslash, URLs are forward slash)